playground/frontend/playground_components/lib/src/widgets/output/result_tab_content.dart

```diff
@@ -46,21 +48,32 @@ class _ResultTabContentState extends State<ResultTabContent> {
     return UnreadClearer(
       controller: widget.playgroundController.codeRunner.unreadController,
       unreadKey: UnreadEntryEnum.result,
-      child: AnimatedBuilder(
-        animation: widget.playgroundController.codeRunner,
-        builder: (context, child) => SingleChildScrollView(
-          controller: _scrollController,
-          child: Scrollbar(
-            thumbVisibility: true,
-            trackVisibility: true,
+      child: ColoredBox(
+        color: Theme.of(context).backgroundColor,
+        child: AnimatedBuilder(
+          animation: widget.playgroundController.codeRunner,
+          builder: (context, child) => SingleChildScrollView(
             controller: _scrollController,
-            child: Padding(
-              padding: const EdgeInsets.all(BeamSizes.size16),
-              child: AnimatedBuilder(
-                animation: widget.playgroundController.resultFilterController,
-                builder: (context, child) => SelectableText(
-                  _getText(),
-                  style: ext.codeRootStyle,
+            child: Scrollbar(
+              thumbVisibility: true,
+              trackVisibility: true,
+              controller: _scrollController,
+              child: Padding(
+                padding: const EdgeInsets.all(BeamSizes.size16),
+                child: AnimatedBuilder(
+                  animation: widget.playgroundController.resultFilterController,
+                  builder: (context, child) {
+                    _codeController.text = _getText();
```

Comment:
Never set controller values in `build()`.
Some controllers will through exception on that phase.
In this particular case the selection will be lost on each build.